Source/WebCore/Modules/mediastream/gstreamer/GStreamerRtpTransceiverBackend.cpp

```diff
@@ -135,16 +135,35 @@ static inline WARN_UNUSED_RETURN ExceptionOr<GstCaps*> toRtpCodecCapability(cons
         }
     }
 
+    if (msid)
+        gst_caps_set_simple(caps, "a-msid", G_TYPE_STRING, msid, nullptr);
+
     GST_DEBUG("Codec capability: %" GST_PTR_FORMAT, caps);
     return caps;
 }
 
+static GUniquePtr<char> getMsidFromCurrentCodecPreferences(GstWebRTCRTPTransceiver* transceiver)
+{
+    GRefPtr<GstCaps> currentCaps;
+    GUniquePtr<char> msid;
+    g_object_get(transceiver, "codec-preferences", &currentCaps.outPtr(), nullptr);
+#ifndef GST_DISABLE_GST_DEBUG
```

Comment:
This is useful mostly if the debug log implies some string copy or allocation usually, which is not the case here, so  the ifdef can be removed imho

Reply:
Makes sense. Done :)